tests/Predis/ClientTest.php

```diff
@@ -618,17 +639,6 @@ public function testCreateClientWithConnectionFromAggregateConnection()
         $this->assertSame($client->getOptions(), $clientNode02->getOptions());
     }
 
-    /**
-     * @group disconnected
-     */
-    public function testGetClientForReturnsInstanceOfSubclass()
-    {
-        $nodes = array('tcp://host1?alias=node01', 'tcp://host2?alias=node02');
-        $client = $this->getMock('Predis\Client', array('dummy'), array($nodes), 'SubclassedClient');
-
-        $this->assertInstanceOf('SubclassedClient', $client->getClientFor('node02'));
-    }
```

Comment:
Any reason why this test has been removed instead of getting fixed for PHPUnit 8.x?